automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/library/DataLibraryPage.scala

```diff
@@ -79,6 +83,35 @@ class DataLibraryPage(implicit webDriver: WebDriver) extends BaseFireCloudPage
     tags.getTags
   }
 
+  def doTagsSearch(tags: String*): List[Map[String, String]] = {
+    clearTags()
+    tags.foreach { tag =>
+      tagsSearchField.setText(tag)
```

Comment:
It might be useful to make this a Component to use it in other places (like the workspace summary page), but it doesn't necessarily need to be in this PR.

Reply:
Looks like a Tags component already exists in `org/broadinstitute/dsde/firecloud/component/Tags.scala`